bin/benchmark_pydy_code_gen.py

```diff
@@ -5,6 +5,8 @@
 
 # external libraries
 from numpy import hstack, ones, pi, linspace, array, zeros, zeros_like
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
It looks like it is always available:
http://matplotlib.1069221.n5.nabble.com/Any-available-backend-for-non-root-users-td38528.html